lib/ansible/modules/files/unarchive.py

```diff
@@ -26,6 +26,7 @@
      - By default, it will copy the source file from the local system to the target before unpacking.
      - Set C(remote_src=yes) to unpack an archive which already exists on the target.
      - For Windows targets, use the M(win_unzip) module instead.
+     - If checksum validation is desired, use M(get_url) or M(uri) instead to fetch the file.
```

Comment:
Would it make this more clear to also indicate that `unarchive` should then be invoked with `remote_src=True` in this scenario?

Reply:
Yes it does. Update MR.